client/components/pack_table/DeletePackItemModal.js

```diff
@@ -4,18 +4,33 @@ import { useDispatch } from "react-redux";
 import { deletePackItem } from "../../store/packsStore";
 import { CustomModal } from "../modal";
 import { deleteGlobalItem } from "../../store/globalItemsStore";
-
+import { isConnected } from "~/utils/netInfo";
```

Comment:
We should probably just have a network slice of store right ? Otherwise we will be repeating this logic a lot.